mos-platform/atari8-xegs/link.ld

```diff
@@ -0,0 +1,333 @@
+  .bank10    : { *(.bank10    .bank10.*)    } >bank10
+  .bank11    : { *(.bank11    .bank11.*)    } >bank11
+  .bank12    : { *(.bank12    .bank12.*)    } >bank12
+  .bank13    : { *(.bank13    .bank13.*)    } >bank13
+  .bank14    : { *(.bank14    .bank14.*)    } >bank14
+  .bank15    : { *(.bank15    .bank15.*)    } >bank15
+  .bank16    : { *(.bank16    .bank16.*)    } >bank16
+  .bank17    : { *(.bank17    .bank17.*)    } >bank17
+  .bank18    : { *(.bank18    .bank18.*)    } >bank18
+  .bank19    : { *(.bank19    .bank19.*)    } >bank19
+  .bank20    : { *(.bank20    .bank20.*)    } >bank20
+  .bank21    : { *(.bank21    .bank21.*)    } >bank21
+  .bank22    : { *(.bank22    .bank22.*)    } >bank22
+  .bank23    : { *(.bank23    .bank23.*)    } >bank23
+  .bank24    : { *(.bank24    .bank24.*)    } >bank24
+  .bank25    : { *(.bank25    .bank25.*)    } >bank25
+  .bank26    : { *(.bank26    .bank26.*)    } >bank26
+  .bank27    : { *(.bank27    .bank27.*)    } >bank27
+  .bank28    : { *(.bank28    .bank28.*)    } >bank28
+  .bank29    : { *(.bank29    .bank29.*)    } >bank29
+  .bank30    : { *(.bank30    .bank30.*)    } >bank30
+
+  .bank31    : { *(.bank31    .bank31.*)    } >bank31
+  .bank32    : { *(.bank32    .bank32.*)    } >bank32
+  .bank33    : { *(.bank33    .bank33.*)    } >bank33
+  .bank34    : { *(.bank34    .bank34.*)    } >bank34
+  .bank35    : { *(.bank35    .bank35.*)    } >bank35
+  .bank36    : { *(.bank36    .bank36.*)    } >bank36
+  .bank37    : { *(.bank37    .bank37.*)    } >bank37
+  .bank38    : { *(.bank38    .bank38.*)    } >bank38
+  .bank39    : { *(.bank39    .bank39.*)    } >bank39
+  .bank40    : { *(.bank40    .bank40.*)    } >bank40
+  .bank41    : { *(.bank41    .bank41.*)    } >bank41
+  .bank42    : { *(.bank42    .bank42.*)    } >bank42
+  .bank43    : { *(.bank43    .bank43.*)    } >bank43
+  .bank44    : { *(.bank44    .bank44.*)    } >bank44
+  .bank45    : { *(.bank45    .bank45.*)    } >bank45
+  .bank46    : { *(.bank46    .bank46.*)    } >bank46
+  .bank47    : { *(.bank47    .bank47.*)    } >bank47
+  .bank48    : { *(.bank48    .bank48.*)    } >bank48
+  .bank49    : { *(.bank49    .bank49.*)    } >bank49
+  .bank50    : { *(.bank50    .bank50.*)    } >bank50
+  .bank51    : { *(.bank51    .bank51.*)    } >bank51
+  .bank52    : { *(.bank52    .bank52.*)    } >bank52
+  .bank53    : { *(.bank53    .bank53.*)    } >bank53
+  .bank54    : { *(.bank54    .bank54.*)    } >bank54
+  .bank55    : { *(.bank55    .bank55.*)    } >bank55
+  .bank56    : { *(.bank56    .bank56.*)    } >bank56
+  .bank57    : { *(.bank57    .bank57.*)    } >bank57
+  .bank58    : { *(.bank58    .bank58.*)    } >bank58
+  .bank59    : { *(.bank59    .bank59.*)    } >bank59
+  .bank60    : { *(.bank60    .bank60.*)    } >bank60
+  .bank61    : { *(.bank61    .bank61.*)    } >bank61
+  .bank62    : { *(.bank62    .bank62.*)    } >bank62
+
+ /* "SIZE=nnn"; used when testing linker scripts, makes hexdump useful */
```

Comment:
The blessed way to do this is to generate a linker map file: -Wl,`-Map=<mapfile>`. One can also obtain size information using llvm-readelf on the .elf file produced by the link; accordingly, it seems better to get the 8 bytes back and have a more typical linker script.

Reply:
This is a hack I was using when testing/debugging the linker script, `hexdump -C` / `hd` is useful when the images are small/sparse to eyeball what's going on.  I will remove this.